boards/nrf52840-dongle/Makefile.dep

```diff
@@ -0,0 +1,25 @@
+ifneq (,$(filter saul_default,$(USEMODULE)))
+  USEMODULE += saul_gpio
+endif
+
+# It'd be nice to pull this in on any stdin dependency and not just shell, but
+# that's the common one, and by the time shell has pulled in stdin, the default
+# stdio_uart has already been set.
+#
+# Maybe have a shell_default like netdev_default?
+ifneq (,$(filter shell,$(USEMODULE)))
+  ifeq (,$(filter stdio_%,$(USEMODULE)))
+    USEMODULE += auto_init_usbus
+    USEMODULE += stdio_cdc_acm
+  endif
+endif
+
+# This will be an additional default netdev to the one set in
+# common/nrf52/Makefile.dep. NETIF_NUMOF will be bumped in the
+# Makefile.include.
+ifneq (,$(filter gnrc_netdev_default,$(USEMODULE)))
```

Comment:
Shouldn't this be `netdev_default`?

Reply:
Probably -- I'd read up on the details (oh that'd be #7094), but anyway with the rollback of the default netdev addition (https://github.com/RIOT-OS/RIOT/pull/13146#discussion_r367882510), this is moot here.